lib/matplotlib/dates.py

```diff
@@ -1,6 +1,6 @@
 """
 Matplotlib provides sophisticated date plotting capabilities, standing on the
-shoulders of python :mod:`datetime` and the add-on module :mod:`dateutil`.
+shoulders of python :mod:`datetime` and the add-on module dateutil_.
```

Comment:
Could we add the dateutil intersphinx and leave the markup?

Reply:
I do not fully understand exactly how this works, but it was a way to get a proper link. Maybe there are better ways.